chia/wallet/wallet_state_manager.py

```diff
@@ -761,15 +761,15 @@ async def spend_clawback_coins(self, clawback_coins: Dict[Coin, ClawbackMetadata
                 if self.main_wallet.secret_key_store.secret_key_for_public_key(derivation_record.pubkey) is None:
                     await self.main_wallet.hack_populate_secret_key_for_puzzle_hash(derivation_record.puzzle_hash)
                 amount = uint64(amount + coin.amount)
+                # Remove the clawback hint since it is unnecessary for the XCH coin
+                memos: List[bytes] = [] if len(incoming_tx.memos) == 0 else incoming_tx.memos[0][1][1:]
```

Comment:
```suggestion
                memos: List[bytes] = incoming_tx.memos[0][1][1:]
```

Is there any scenario how the memos could be empty here?

Reply:
It shouldn't be empty, just in case of unexpected things happen.